public/app/plugins/panel/geomap/migrations.ts

```diff
@@ -100,23 +104,38 @@ function asNumber(v: any): number | undefined {
 }
 
 export const mapMigrationHandler = (panel: PanelModel): Partial<GeomapPanelOptions> => {
-  const pluginVersion = panel?.pluginVersion;
-  if (pluginVersion?.startsWith('8.1') || pluginVersion?.startsWith('8.2') || pluginVersion?.startsWith('8.3')) {
-    if (panel.options?.layers?.length > 0) {
+  const pluginVersion = panel?.pluginVersion ?? '';
+
+  // before 8.3, only one layer was supported!
```

Comment:
Do we have any external places for documenting versioning of panels / plugins and their related constraints? I could see this becoming challenging to manage at more scale